### ISSUE TITLE
On latest version of chrome-beta, favicons do not load until the tab is put in focus

### ISSUE THREAD

**Issue by christopherreay** [deanoemcke/thegreatsuspender#883]
recent:///4ab432bd0f0f0ffdb1543f6f5c84e3ecPlease complete the following information when submitting a feature request or bug report.
 * **Extension version:**
   * The Great Suspender v6.30
 * **Browser name & version:**
   * Chrome: Version 72.0.3626.28 (Official Build) beta (64-bit)
 * **Operating system & version:**
   * Linux 4.15.0-36-generic #39~16.04.1-Ubuntu SMP Tue Sep 25 08:59:23 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

And please also do a search for your request/bug before create a new one thanks!
* I searched for "favicon" in open issues and could find nothing relevant.

Umm. yes, as it says in the title. If I ctrl-PgDn through all the tabs they all load the favicons, rather than the greatSuspended's beautiful face. This used to not happen on the previous version of chrome I was using.

recent:///4ab432bd0f0f0ffdb1543f6f5c84e3ec

this "recent:////" thing is supposed to be a screenshot... tell me if it works... I just dragged and dropped it

recent:///4ab432bd0f0f0ffdb1543f6f5c84e3ec

no idea

[image on my server](http://christopherreay.com/fileStorage/tooMuchSuspenderFace.png)

**Comment by Jollfye** [comment]
I guess it's a known and already fixed issue in development version of TGS, see for example #860, issue itself is a duplicate of #719.

**Comment by christopherreay** [comment]
Sorry I should have checked closed issues. Whats the process for closing issues? Perhaps there could be a tag for issues which is "fixed but not released yet".

Is there any way to install the latest development version of TGS? or is that inadvisable?

This extension is one of the most categorically useful core parts of my working process. I have about 30 browser profiles (different projects) and probably 2,500 tabs stored in all of them. Works like a treat. I shall donate ... might have already...

**Comment by Jollfye** [comment]
@christopherreay, well there're ways to install latest dev version of TGS extension, I was talking about it [here](https://github.com/deanoemcke/thegreatsuspender/issues/879#issuecomment-468986868), [here](https://github.com/deanoemcke/thegreatsuspender/issues/849#issuecomment-460075327) (near the end, less detailed) and more detailed [here](https://github.com/deanoemcke/thegreatsuspender/issues/839#issuecomment-458312906) (tho now I decided that to remove old version of TGS is unnecessary to try the latest one, just disabling may be enough as I described in my first "here" here). Also considering that u have that much Chrome profiles and tabs I guess it may be quite hard to backup (export to file) all of them profile by profile so I would suggest u to backup (copy/paste somewhere else) ur whole C:\Users\User\AppData\Local\Google\Chrome profiles and data folder (that's for Windows 10, look for analog on Linux) before u start any updating/trying latest dev version of The Great Suspender in case smth goes wrong and ur tabs lost so u could just delete broken folder and restore it all with original one while Chrome is exited. Then go to profile with least tabs and work with it to try latest dev version of TGS as I suggested in my first "here". And here I'm not sure how disabling old version will affect inactive (other) Chrome profiles except current. Logically speaking it shouldn't (coz they're completely separate each with its own instances of extensions) but yeah idk as I have just one Chrome profile so never tried. In current Chrome profile disabling of old TGS will disappear all suspended tabs so unsuspend them before. Then try install latest dev version. But here I also may see a mess coz of difference in extension folder names in Chrome, Web Store TGS may have name gbcokmhikpgllchhjdjdklejaicdbhmg or anything else similar but unpacked latest one may have smth different but also similar so all suspended tabs in other inactive Chrome profiles may have old folder name in their addresses so those tabs won't work with unpacked latest TGS extension. Soo as I can see if u would have one Chrome profile it might be easy to try latest dev version but as soon as u have more it may be quite messy (or it may be not coz all profiles r separate, in every profile its own TGS, just unsuspend all tabs before disable and all ok). Sooo here I suggest u to wait for 7.1 release of TGS in Chrome Web Store and when it comes out to backup ur whole Chrome data folder before any update. And while u waiting u could install as unpacked TGS latest dev version on ur most used Chrome profiles by scheme: unsuspend all, disable, install unpacked, suspend all. Hmm and here I'm thinking again that every Chrome profile is actually like a lone unit and they're not connected so in every Chrome profile u may be have its own TGS version installed soo that scheme above could be repeated for every of ur Chrome profiles but that may be too hard and unnecessary coz u have so much profiles. So update just a few with dev version then wait for update to come out and update all others from Web Store. Then to go back to Web Store version for those with dev one use the scheme: unsuspend all tabs, disable dev version (when there's no suspended tabs left coz they may disappear at the moment), enable old Web Store version, update it to when release 7.1 by clicking Update in Chrome Extensions to manually check for update and update ur installed extensions, suspend all tabs (maybe it needs to be done before update and after enable of old version, not sure) and that's may be it and that for every profile with dev version, then restart Chrome and I guess that's finally it. Why try unpacked dev version on less of ur Chrome profiles? Coz Web Store version can be auto-updated + manually updated with Update button when dev version can be updated only manually by replacing files with new ones from downloaded zip. So here u have it, proceed with caution and backups.

**Comment by greatsuspender** [comment]
Duplicate of: #719
